src/components/fields/date-field/date-field.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import requiredIf from 'react-required-if';
+import Constraints from '../../constraints';
+import Spacings from '../../spacings';
+import FieldLabel from '../../field-label';
+import DateInput from '../../inputs/date-input';
+import createSequentialId from '../../../utils/create-sequential-id';
+import FieldErrors from '../../field-errors';
+import filterDataAttributes from '../../../utils/filter-data-attributes';
+
+const sequentialId = createSequentialId('date-field-');
+
+const hasErrors = errors => errors && Object.values(errors).some(Boolean);
+
+class DateField extends React.Component {
+  static displayName = 'DateField';
+
+  static isEmpty = DateInput.isEmpty;
```

Comment:
Didn't know we do this for all fields.

Reply:
It felt like a good idea to take the burden of determining the empty state away from the consumers. We don't have this functionality anywhere yet, but seems like it would be useful for all inputs & fields.

Reply:
Sure. I am just arguing from a consumer perspective. My intuition would have "told" me to use the respective `*Input` for the `isEmpty` check and not the field itself. Not saying to change it. Other folks might have different expectations.

Reply:
Yeah we could do that as well. having to redeclare the statics on the fields, and having to document the statics on the field again is a bit of overhead for the ui-kit anyways. It might make sense to educate consumers of these components to use the statics of the underlying inputs instead.

Thanks for pointing it out 🥇 

cc @montezume